docs/learn/learn-guides-bounties.md

```diff
@@ -80,6 +80,21 @@ extrinsic. More information about when to do this can be found on the main
 [Bounties page](https://polkadot.js.org/apps/#/bounties). Only after the curators claim their
 candidacy the bounty will show as "active" on the main Bounty page.
 
+:::info Curator assignment must execute after bounty funding
+
+Curator assignment must take place after the bounty has been funded, and approved bounties can only
+be funded after a spend period has been completed. A curator assignment referendum cannot be
+executed during the same spend period as the bounty funding.
+
+If your bounty has been awarded, it will be funded at the end of the current spending period. You
```

Comment:
This first sentence will be unnecessary, if it is clarified already before.

Reply:
deleted